agreement/pseudonode_test.go

```diff
@@ -520,21 +519,21 @@ func TestPseudonodeNonEnqueuedTasks(t *testing.T) {
 	for i := 0; i < pseudonodeVerificationBacklog*2; i++ {
 		ch, err = pb.MakeProposals(context.Background(), startRound, period(i))
 		if err != nil {
-			require.ErrorAs(t, errPseudonodeBacklogFull, &err)
+			require.ErrorIs(t, err, errPseudonodeBacklogFull)
```

Comment:
This never inspected the real error; it just tried to call `errors.As` to assign the constant `errPseudonodeBacklogFull` into the local `err` variable.